### ISSUE TITLE
Audit FreeBMD1

### ISSUE THREAD

**Issue by PatReynolds** [FreeUKGen/FreeBMD2#4]
List all the pages / functionality

**Comment by richpomfret** [comment]
Emailed Barrie/Allan to see if we can get access to any parts of the site which may be restricted by user type.

**Comment by DeniseColbert** [comment]
https://drive.google.com/open?id=17OxzUpRAJxfcmb0uLMupK471q64Dr9dcPdKFdKVIrvk

**Comment by PatReynolds** [comment]
With @richpomfret for technical checking

**Comment by richpomfret** [comment]
Happy with this after an initial review - I'd like to spend a bit more time going through it and it is something we can also ask Barrie to take a look at.

**Comment by mburnham-1** [comment]
Hi Rich,If the next conference call is this Wednesday, I regret I shan't be able to attend as I have a Hospital appointment that morning.I shall be back to the web page shortly.Best wishes to all for a great 2019.Mike
    On Wednesday, 2 January 2019, 10:52:32 GMT, Rich Pomfret <notifications@github.com> wrote:  
 
 
Happy with this after an initial review - I'd like to spend a bit more time going through it and it is something we can also ask Barrie to take a look at.

**Comment by richpomfret** [comment]
Extremely useful having this document - I wonder what remains to be done/added to it? I'd like to get Allan/Barrie to take a look once we think it is complete, they can then let us know if anything has been missed. This can then help us to build our structured development plan (order of what pages we want to develop alongside the Search page).

**Comment by richpomfret** [comment]
Secondary set now ready for review.

**Comment by richpomfret** [comment]
On hold while Mike is away on his trip.

**Comment by richpomfret** [comment]
Closing - can create new story if we need.
